lib/src/element/text.cpp

```diff
@@ -801,7 +801,13 @@ namespace cycfi { namespace elements
       if (_select_end == -1)
          return;
 
-      auto info = caret_info(ctx, &get_text()[_select_end]);
+      basic_text_box::caret_metrics info;
+      if (_select_end >= get_text().size()) {
```

Comment:
For consistency with the code base, can we get rid of the braces?

Reply:
ok，Didn't notice